resolwe_bio/processes/workflows/rnaseq.yml

```diff
@@ -229,3 +229,76 @@
           mode: union
           stranded: '{{input.stranded}}'
           id_attribute: '{{input.id_attribute}}'
+
+- slug: dss-rnaseq
+  name: RNA-Seq for biologists
```

Comment:
Either use title case "RNA-Seq for Biologists" or sentence case "RNA-seq for biologists".